Tasks/DeployVisualStudioTestAgent/TestAgentInstall.ps1

```diff
@@ -40,7 +40,21 @@ function Install-Product($SetupPath, $UserName, $Password, $ProductVersion, $Arg
 
 		if(-not ($exitCode -eq 0 -or $exitCode -eq 3010 -or $exitCode -eq 3015 -or $exitCode -eq 1641))
```

Comment:
when there is a pending restart required what is the exit code? we should land here only in such cases